### PR TITLE
[Feature] Support multiple HDFS in Firestorm (Part2 for Shuffle Server & Driver)

### PR DESCRIPTION
**Pull request by colinmjj**
### What changes were proposed in this pull request?
The PR is target to support multiple HDFS with RSS cluster, and it's the 2nd part for Shuffle Server & Driver

### Why are the changes needed?
To release I/O pressure when with multiple HDFS cluster

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
new UT added

### CODE REVIEW

File: integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java [hunk 1]
```diff
@@ -112,4 +129,23 @@ protected static void createAndStartServers(
...
+    Thread.sleep(2000);
+    FileWriter fileWriter = new FileWriter(cfgFile);
+    PrintWriter printWriter = new PrintWriter(fileWriter);
+    for (Map.Entry<String, String> entry : dynamicConf.entrySet()) {
+      printWriter.println(entry.getKey() + " " + entry.getValue());
+    }
+    printWriter.flush();
+    printWriter.close();
+  }
 }
```
Comment (duanmeng) [inline integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java:151]: empty space

File: integration-test/common/src/test/java/com/tencent/rss/test/MultiStorageFaultToleranceTest.java [hunk 2]
```diff
@@ -36,24 +48,15 @@
...
-import java.util.Map;
-import java.util.Set;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
 public class MultiStorageFaultToleranceTest extends ShuffleReadWriteBase {
   private ShuffleServerGrpcClient shuffleServerClient;
+  private static String REMOTE_STORAGE = HDFS_URI + "rss/multi_storage_fault";
```
Comment (duanmeng) [inline integration-test/common/src/test/java/com/tencent/rss/test/MultiStorageFaultToleranceTest.java:58]: private static final?

Comment (colinmjj) [reply integration-test/common/src/test/java/com/tencent/rss/test/MultiStorageFaultToleranceTest.java:58]: ok

**Comment by jerqi** [comment]
I think we should add a integration test that we test the coordinator assgin HDFS storage path according to their use times.

**Comment by jerqi** [comment]
I think we should add a integration test that we test the coordinator assgin HDFS storage path according to their use times.

**Comment by colinmjj** [comment]
> 

It should be tested in FetchClientConfTest.testFetchRemoteStorage already with driver & coordinator part.

**Comment by jerqi** [comment]
OK, no another suggestion.

**Review by jerqi — APPROVED** [review]

**Pull request merged**